gnome-initial-setup/pages/account/gis-account-page.c

```diff
@@ -1403,6 +1403,9 @@ gis_account_page_init (GisAccountPage *page)
 void
 gis_prepare_account_page (GisDriver *driver)
 {
+  if (gis_driver_is_skipped_page (driver, PAGE_ID))
+    return;
```

Comment:
The problem with the skip file in the driver is that it gets considered **before** the GisPage instances are created, which is before we give the branding-welcome page a chance to load the data from the configuration file, and parse it, so I'd rather not use that.

That said, I did it this way because I wanted to prevent the page from being added to the driver at all, but if hiding it in the constructor works too that's fine as well for me.